db.json

```diff
@@ -6495,20 +7631,21 @@
   "application/xliff+xml": {
     "source": "iana",
     "compressible": true,
-    "extensions": ["xlf"]
+    "extensions": ["xlf","xliff"]
```

Comment:
https://www.iana.org/assignments/media-types/application/xliff+xml only lists xlf

But https://github.com/search?q=path%3A*.xliff&type=code shows a lot of positive hits